website/content/docs/concepts/cpu.mdx

```diff
@@ -0,0 +1,317 @@
+accessing system memory.
+
+
+[![NUMA](/img/nomad-numa.png)](/img/nomad-numa.png)
+
+The simplified example machine above has the following topology
+- 2 physical CPU sockets
+- 4 system memory banks, 2 per socket
+- 8 physical cpu cores (4 per socket)
+- 2 logical cpu cores per physical core
+- 4 PCI devices, 1 per memory bank
+
+### optimizing performance
+
+Any OS process scheduled on core X where X belongs to  memory node M will take
+longer to access any memory node that is not M.
+
+Using the example above if a task is scheduled on Core 0, accessing memory in
+Mem 1 might take 20% longer than accessing memory in Mem 0, and accessing memory
+in Mem 2 might take 300% longer. The extreme differences are due to various
+physical hardware limitations. A core accessing memory in its own NUMA node is
+optimal. Programs which perform a high throughput of reads or writes to/from
+system memory will have their performance substantially hindered by not optimizing
+their spatial locality iwth regard to the systems NUMA topology.
+
+### SLIT tables
+
+Any modern machine will define System Locality Distance Information (SLIT)
+tables in its firmware. These tables are understood and made referenceable
+by the Linux kernel. There are two key pieces of information provided by SLIT
+tables
+- Which CPU cores belong to which NUMA nodes
+- The penalty incurred for accessing each NUMA node from a core in every other
+NUMA node
+
+The `lscpu` command can be used to describe the Core associativity on a machine.
+For example on an `r6a.metal` EC2 instance:
+
+```shell-session
+$ lscpu | grep NUMA
+NUMA node(s):           4
+NUMA node0 CPU(s):      0-23,96-119
+NUMA node1 CPU(s):      24-47,120-143
+NUMA node2 CPU(s):      48-71,144-167
+NUMA node3 CPU(s):      72-95,168-191
+```
+
+And the associated performance degredations are available via `numactl`
+
+```shell-session
+$ numactl -H
+available: 4 nodes (0-3)
+...
+node distances:
+node   0   1   2   3
+  0:  10  12  32  32
+  1:  12  10  32  32
+  2:  32  32  10  12
+  3:  32  32  12  10
+```
+
+These SLIT table "node distance" values are presented as approximate relative
+ratios. The value of 10 represents an optimal situation where a memory access
+is occuring from a CPU that is part of the same NUMA node. A value of 20 for
+example would indicate a 200% performance degredation, 30 for 300%, etc.
+
+### node attributes
+
+A Nomad client will fingerprint the machine's NUMA topology and export the
+core associativity as node attributes. This data can provide a Nomad operator
+a better understanding of when it might be useful to make use of NUMA aware
+scheduling for certain workloads.
+
+```
+numa.node.count       = 4
+numa.node0.cores      = 0-23,96-119
+numa.node1.cores      = 24-47,120-143
+numa.node2.cores      = 48-71,144-167
+numa.node3.cores      = 72-95,168-191
+```
+
+## NUMA aware scheduling
+
+Starting in Nomad Enterprise 1.7, Nomad is capable of scheduling tasks in a way
```

Comment:
```suggestion
Nomad Enterprise is capable of scheduling tasks in a way
```

With versioned docs I think we can avoid referencing specific versions in docs so things don't feel state.